src/staff-handbook/policies-and-procedures/index.md

```diff
@@ -5,3 +5,10 @@ last_reviewed_at: ""
 ---
 
 All Information Security Management System (ISMS) policies can be found in the shared ISMS folder in Google Drive.
+
+* <a href="/staff-handbook/policies-and-procedures/disciplinary-procedure/">Disciplinary procedure</a>
```

Comment:
Possibly I should have done this in Markdown rather than HTML 🤦🏻

Reply:
That would be good!